src/app/lib/utilities/ichefURL/index.js

```diff
@@ -11,40 +31,33 @@ const buildPlaceholderSrc = (src, resolution) => {
   const urlParts = imageSrc.replace(/https?:\/\//g, '').split('/');
   const [domain, mediaType, imgService, ...remainingUrlParts] = urlParts;
   const remainingUrlPartsWithoutResolution = remainingUrlParts.slice(1);
+  const webpImgUrl = `${remainingUrlPartsWithoutResolution}.webp`;
```

Comment:
You are right this code looks wrong now I look at it again 😂 Maybe this actually shows this code isn't tested! The test that is supposed to be testing the placeholder srcset is this

`  it.each(['mpv', 'pips'])(
    'returns a placeholder image if no image src provided and origin code is %s',
    originCode => {
      expect(getIChefURL({ resolution: 512, originCode })).toEqual(
        'https://ichef.bbci.co.uk/images/ic/512xn/p0b36kgx.png.webp',
      );
    },
  );`

which uses this next:

`  if (originCode === 'mpv' || originCode === 'pips') {
    return buildPlaceholderSrc(locator, resolution);
  }`

and then 

`const buildPlaceholderSrc = (src, resolution) => {
  const imageSrc =
    src || 'https://ichef.bbci.co.uk/images/ic/640xn/p0b36kgx.png';`

but because the test doesn't provide a src, the hard coded url is always used in the test and this just happens to work as there is only one part in the 'remainingUrlParts'.

I shall change this.